tests/extensions/test_kingfisher_files_store.py

```diff
@@ -45,23 +45,27 @@ def test_item_scraped_with_build_file_from_response(sample, path, tmpdir):
 
 
 @pytest.mark.parametrize('sample,path', [
-    (None, os.path.join('test', '20010203_040506', 'file.json')),
-    ('true', os.path.join('test_sample', '20010203_040506', 'file.json')),
+    (None, os.path.join('test', '20010203_040506')),
+    ('true', os.path.join('test_sample', '20010203_040506')),
 ])
 @pytest.mark.parametrize('data', [b'{"key": "value"}', {"key": "value"}])
-def test_item_scraped_with_build_file(sample, path, data, tmpdir):
+@pytest.mark.parametrize('item,expected_file_name', [(File({'file_name': 'file.json', 'encoding': 'iso-8859-1'}),
+                                                      'file.json'),
+                                                     (FileItem({'number': 1, 'file_name': 'file.json'}),
+                                                      'file-1.json')])
+def test_item_scraped_with_file_and_file_item(sample, path, data, tmpdir, item, expected_file_name):
```

Comment:
To avoid confusion with parametrize arguments, I prefer to put pytest fixtures at the end:

```suggestion
def test_item_scraped_with_file_and_file_item(sample, path, data, item, expected_file_name, tmpdir):
```